src/handlers/organizationsHandlers.ts

```diff
@@ -45,13 +45,15 @@ const postOrganization = (
 };
 
 const getOrganizations = (
-	req: AuthenticatedRequest,
+	req: Request,
 	res: Response,
 	next: NextFunction,
 ): void => {
 	const paginationParameters = extractPaginationParameters(req);
 	const proposalParameters = extractProposalParameters(req);
-	const authenticationId = req.user?.authenticationId;
+	const authenticationId = isAuthContext(req)
+		? req.user?.authenticationId
+		: undefined;
```

Comment:
Since we repeat this several times, it might be worth extracting a helper function! `getAuthenticationId(Request) -> string | undefined` or so?

Reply:
That seems reasonable, but this block gets blown away in the normalization PR so I'd prefer to leave it as is for this specific PR and circle back once main reflects the latest and greatest of all these changes.

Reply:
It ended up being more related to the now-merged code / doesn't fit in this PR.

I opened a new one for it: https://github.com/PhilanthropyDataCommons/service/pull/1217